packages/graph/hash_graph/lib/graph/src/shared/provenance.rs

```diff
@@ -9,8 +10,20 @@ use crate::identifier::account::AccountId;
 macro_rules! define_provenance_id {
     ($name:tt) => {
         #[derive(
-            Debug, Copy, Clone, PartialEq, Eq, Hash, PartialOrd, Ord, Serialize, Deserialize,
+            Debug,
+            Copy,
+            Clone,
+            PartialEq,
+            Eq,
+            Hash,
+            PartialOrd,
+            Ord,
+            Serialize,
+            Deserialize,
+            FromSql,
+            ToSql,
```

Comment:
Similar question to why we've added this to `EntityUuid`

Reply:
As discussed offline, I removed the `FromSql` implementation: 335193c558819a6b09279c87a1834e6774588c5b